### ISSUE TITLE
(Voicing) - Please remove repetition of last stop responses at ends of position/distance sliders

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#292]
In https://github.com/phetsims/gravity-force-lab-basics/issues/291#issuecomment-888333627, we noted a voicing issue for the move spheres slider interactions.

When a learner hits the edge, and tries to go beyond the min or max position, the last-stop-changing-force response repeats.

**Please make it so, nothing voices after a learner hits the last stop.**

Note that this is not an issue for the mass interactions. Nothing voices when trying to go beyond the max or min mass.

**Additional comments:**
Hopefully, the visual design, the bump sounds and the "last stop" phrase indicate to learners they cannot explore further in that direction and getting not further feedback should make it clear that no further changes are happening.

In addition learners are able to access current state information by investigating the spheres, accessing the reading blocks (force or distance), and/or by playing the current details.

**Comment by jessegreenberg** [comment]
Done in the above commit. Will prepare a new version for review shortly.

**Comment by jessegreenberg** [comment]
@terracoda can you please review in this version? https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-dev.36/phet/gravity-force-lab-basics_en_phet.html

**Comment by terracoda** [comment]
Sounds great @jessegreenberg.
No repetition of force response, that is nothing after last stop. Thanks!
